site/src/pages/docs/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import LandingPage from '@site/src/components/LandingPage.js';
+
+const docsList = [
```

Comment:
Why is this list of products different from the one on the landing page? I.e., when you arrive at the docsite you see these:
![image](https://user-images.githubusercontent.com/3445445/185630883-8a7ee976-0e5b-458e-a099-f9514dd5b797.png)


But when you navigate to docs you see the list below which includes Account and has the other ones show up in a different order? There's probably a good reason for leaving Account out that I'm just unaware of lol

![image](https://user-images.githubusercontent.com/3445445/185630762-2373ef07-d4ab-4090-b035-85549c16a431.png)

Reply:
The subtitle for both pages probably could be the same as well. I just felt like it was kind of weird to go from the landing page to the docs page where it was almost the same but not

Reply:
The account link was left out of the homepage because its technically not a separate product, but it exists in the docs page because we have separate docs for account management. We talked about it yesterday and now that the docs landing page mimics the others, it doesn't make any sense to have it differ from the homepage. This PR might sit for a little bit pending approval from @proy-bw and the state of the `startStream` verb, so for now its alright that they're so similar. DX-2791 is set to change the current home page into a splash/marketing page and then the docs landing page will probably take the content from the current home page and add the account icon.